repo_files/documentations/radar_chart.md

```diff
@@ -1,73 +1,70 @@
 
 
 ### RadarChartData
-|PropName		|Description	|default value|
-|:---------------|:---------------|:-------|
-|dataSets| list of [RadarDataSet ](#RadarDataSet) that is shown on the radar chart|[]|
+|PropName     |Description   |default value|  
+|:---------------|:---------------|:-------|  
+|dataSets| list of [RadarDataSet ](#RadarDataSet) that is shown on the radar chart|[]|  
 |radarBackgroundColor| This property fills the background of the radar with the specified color.| Colors.transparent|
-|radarBorderData| shows a border for radar chart|BorderSide(color: Colors.black, width: 2)|
-|getTitle| This function helps the radar chart to draw titles outside the chart.|null|
-|titleTextStyle|TextStyle of the titles|TextStyle(color: Colors.black, fontSize: 12)|
-|titlePositionPercentageOffset|this field is the place of showing title on the RadarChart. The higher the value of this field, the more titles move away from the chart. this field should be between 0 and 1.|0.2|
-|tickCount|Defines the number of ticks that should be paint in RadarChart|1|
-|ticksTextStyle|TextStyle of the tick titles|TextStyle(fontSize: 10, color: Colors.black)|
-|tickBorderData|Style of the tick borders|BorderSide(color: Colors.black, width: 2)|
-|gridBorderData|Style of the grid borders|BorderSide(color: Colors.black, width: 2)|
-|radarTouchData|Handles touch behaviors and responses.|RadarTouchData()|
+|radarShape| the shape of the border and background |RadarShape.circle|  
+|radarBorderData| shows a border for radar chart|BorderSide(color: Colors.black, width: 2)|  
+|getTitle| This function helps the radar chart to draw titles outside the chart.|null|  
+|titleTextStyle|TextStyle of the titles|TextStyle(color: Colors.black, fontSize: 12)|  
+|titlePositionPercentageOffset|this field is the place of showing title on the RadarChart. The higher the value of this field, the more titles move away from the chart. this field should be between 0 and 1.|0.2|  
+|tickCount|Defines the number of ticks that should be paint in RadarChart|1|  
+|ticksTextStyle|TextStyle of the tick titles|TextStyle(fontSize: 10, color: Colors.black)|  
+|tickBorderData|Style of the tick borders|BorderSide(color: Colors.black, width: 2)|  
+|gridBorderData|Style of the grid borders|BorderSide(color: Colors.black, width: 2)|  
+|radarTouchData|Handles touch behaviors and responses.|RadarTouchData()|  
 
 ### RadarDataSet
-|PropName		|Description	|default value|
-|:---------------|:---------------|:-------|
-|dataEntries|Each RadarDataSet contains list of [RadarEntries ](#RadarEntry) that is shown in RadarChart.|[]|
-|fillColor|fills the DataSet with the specified color.|Colors.black12|
-|borderColor|Paint the DataSet border with the specified color.|Colors.blueAccent|
-|borderWidth|defines the width of [RadarDataSet](#RadarDataSet) border.|2.0|
-|entryRadius|defines the radius of each [RadarEntries ](#RadarEntry).|5.0|
+|PropName     |Description   |default value|  
```

Comment:
This file changed a lot. I think we only need to update a single line (add the new property `radarShape`).
It might be changed by your IDE.
Please revert all changes except the line we need. (I suggest you to update it with a simple text editor instead of directly in your IDE to prevent these changes)

Reply:
Sorry, I made my changes I bit too quickly, and didn't check the diff.
I edited it with slackedit and I think it added 2 spaces at the end of each line to be sure to encode 'return to line'.
Anyway, I refactored, should be ok now :sweat_smile:

Reply:
No worries, I totally understand :)))
I checked and It's okay now.
Thanks for your quick change.

Reply:
No worries, I totally understand :)))
I checked and It's okay now.
Thanks for your quick change.